### PR TITLE
Modified our team page to the current version of EvalAI

### PR DESCRIPTION
**Pull request by Sanji515**
**Screenshot:**

Screenshot from 2019-06-03 01-15-37

### CODE REVIEW

File: src/app/components/our-team/our-team.component.html [hunk 1]
```diff
@@ -1,24 +1,24 @@
 <app-header-static></app-header-static>
 <div class="container-top">
 	<div class="our-team-container ">
-	<div class="our-team-title">
-		{{coreTeamType}}
-	</div>
+        <div class="our-team-title">
```
Comment (Shekharrajak) [inline src/app/components/our-team/our-team.component.html:4]: You can install some extension in  your editor to automatically make proper alignment when you save the file.

File: src/app/components/our-team/our-team.component.scss [hunk 2]
```diff
@@ -87,22 +88,22 @@
...
+        @include box-shadow(0px, 0px, 5px, 0px, $overlay-light);
       }
   }
 
   .team-bg {
       position: absolute;
       top: -40px;
       left: -50px;
-      background: $blue-overlay-dark;
+      background: #252833;;
```
Comment (Shekharrajak) [inline src/app/components/our-team/our-team.component.scss]: Change the variable value `blue-overlay-dark`

File: src/app/components/our-team/our-team.component.scss [hunk 3]
```diff
@@ -150,8 +160,16 @@
       position: absolute;
       width: 100%;
       top: 0;
-      left: 0;
+      left: -15px;
```
Comment (Shekharrajak) [inline src/app/components/our-team/our-team.component.scss]: Can we make it relative to the device width?

**Review by Shekharrajak — COMMENTED** [review]
Please make sure it works in different devices.

File: src/app/components/our-team/our-team.component.scss (hunk 2 above)
Comment (Sanji515) [reply src/app/components/our-team/our-team.component.scss]: @Shekharrajak done

File: src/app/components/our-team/our-team.component.html (hunk 1 above)
Comment (lunayach) [reply src/app/components/our-team/our-team.component.html:4]: Yes, and if you use Py-Charm or VSCode, you could easily auto-format.

File: src/app/components/our-team/our-team.component.scss (hunk 3 above)
Comment (lunayach) [reply src/app/components/our-team/our-team.component.scss]: @Sanji515 is this done?

File: src/app/components/our-team/our-team.component.html (hunk 1 above)
Comment (Sanji515) [reply src/app/components/our-team/our-team.component.html:4]: yeah I use VS code

**Comment by lunayach** [comment]
@Sanji515  Actually, I was unable to see it on the surge and not sure if it's working on all the devices. You may want to add tests and resolve a minor comment.

File: src/app/components/our-team/our-team.component.scss (hunk 3 above)
Comment (Sanji515) [reply src/app/components/our-team/our-team.component.scss]: @lunayach I checked it but it was a very little change from the starting position of card so I keep it as it is and It's looking good in small devices also.

Comment (Sanji515) [reply src/app/components/our-team/our-team.component.scss]: @lunayach @Shekharrajak done

**Comment by Sanji515** [comment]
> You may want to add tests and resolve a minor comment

@lunayach  done. Please review

File: src/app/components/our-team/our-team.component.scss [hunk 4]
```diff
@@ -140,18 +141,34 @@
...
+      }
       h4 {
-          font-size: 16px;
+          font-size: $fs-16;
       }
   }
 
   .team-link-head {
-      padding: 10px 15px;
+      padding: 10px 5px 10px 5px !important;
```
Comment (lunayach) [inline src/app/components/our-team/our-team.component.scss]: I see at many places you using `!important` (in other PR's as well). Generally, it's discouraged to use this and use only if we just can't do without this. Was this the case here?

Comment (Sanji515) [reply src/app/components/our-team/our-team.component.scss]: @lunayach no, previously it was using `padding-left` from some other place, so just to override I used it but it is not required as I've removed that `padding-left`

Removed it now, please review

**Review by lunayach — APPROVED** [review]
I think, this could be merged.

File: src/app/components/our-team/our-team.component.html [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
-                <a ng-if="data.linkedin_url !== 'null'" href="{{data.linkedin_url}}" target="_blank"><i class="fab fa-linkedin"></i></a>
-            </span>
-                        <span>
-                <a ng-if="data.github_url !== 'null'" href="{{data.github_url}}" target="_blank"><i class="fab fa-github"></i></a>
-            </span>
-                        <span class="right link-team-user">
-                <a ng-if="data.personal_website !== 'null'" href="{{data.personal_website}}" target="_blank">Personal Website</a>
-            </span>
+                        <span class="col-md-1 col-sm-1 col-xs-1">
+                            <a ng-if="data.linkedin_url !== 'null'" href="{{data.linkedin_url}}" target="_blank"><i class="fab fa-linkedin"></i></a>
```
Comment (Shekharrajak) [inline src/app/components/our-team/our-team.component.html]: Isn't it still 80+ length. Also, HTML tags are not line by line.

Comment (Sanji515) [reply src/app/components/our-team/our-team.component.html]: @Shekharrajak done, please take a look if it looks good to you

**Pull request merged**